doc/ReleaseNotes/_ReleaseNotes.md

```diff
@@ -47,6 +47,11 @@
  * Multi-selection Check Boxes in ListViewItems are appearing brielfly (https://github.com/nventive/Uno/issues/403)
  * 140721 [Android] FlipView not visible when navigating back to page
  * 138537 [iOS] App freezes after State selection causing infinite load on every subsequent launch
+ * 141932 [Android][FlipView] Application does not transition to the next unanswered question properly.
```

Comment:
We don't need to add canary bugs to release notes :) they were not present in last stable so they're not new fixes.